qutip/states.py

```diff
@@ -291,17 +296,8 @@ def coherent_dm(N, alpha, offset=0, method='operator'):
     but would in that case give more accurate coefficients.
 
     """
-    if method == "operator":
-        psi = coherent(N, alpha, offset=offset)
-        return psi * psi.dag()
-
-    elif method == "analytic":
-        psi = coherent(N, alpha, offset=offset, method='analytic')
-        return psi * psi.dag()
-
-    else:
-        raise TypeError(
-            "The method option can only take values 'operator' or 'analytic'")
+    psi = coherent(N, alpha, offset=offset)
```

Comment:
Shouldn't this pass on the `method` value?

Reply:
Yes :(. I should probably have put WIP in the title given that I just threw something together without checking it. I'll fix everything later tonight.